src/test/java/hudson/plugins/ec2/win/WinConnectionWithFIPSTest.java

```diff
@@ -21,14 +21,6 @@ public void testSelfSignedCertificateNotAllowed() {
         assertThrows(IllegalArgumentException.class, () -> new WinConnection("", "", "", true));
     }
 
-    /**
-     * Using a password with TLS, an {@link IllegalArgumentException} is not expected
-     */
-    @Test
```

Comment:
This test no longer makes sense, any operation mode should fail in FIPS mode

Reply:
I have removed all the tests as they are covered by `testInstantiationWinConnection`